### ISSUE TITLE
[Feature request] git pull --recurse-submodules support

### ISSUE THREAD

**Issue by vadzz-dev** [ForkIssues/TrackerWin#671]
Would be very cool to have such an option in pull window somewhere under stash and reapply local changes

**Comment by felixhirt** [comment]
As far as i understand, the option "File/Preferences/General/Update submodules automatically" should help? It seems to result in a "git pull --verbose --progress --rebase --autostash" followed by a "git submodule update --init --recursive" (with my settings), which to me seems to be the same thing as git pull --recurse-submodules (with the benefit of working if the submodules were never pulled before), i.e. check out the submodule commit referenced in the main repo

**Comment by vadzz-dev** [comment]
@felixhirt thanks it works for me, but this one can anyway be useful for some people that wanna do it just once

**Comment by felixhirt** [comment]
I agree, i don't have the option set because i don't want it to happen always, but occasionally it would be helpful
